libraries/AP_HAL_ChibiOS/hwdef/PixPilot-V6PRO/README.md

```diff
@@ -0,0 +1,352 @@
+## PixPilot-V6PRO Flight Controller
+
+The PixPilot-V6PRO flight controller is sold by a range of resellers listed on the makeflyeasy(http://www.makeflyeasy.com)
+
+## Features
+
+•	STM32H743VIT6 and STM32F103C8T6 microcontroller
+
+•	Three IMUs, two ICM42688-P(SPI), one ICM40605(SPI)
+
+•	internal heater for IMUs temperature control
+
+•	internal Soft Rubber Damping Ball isolation for All interna IMUs
+
+•	Two barometers, BMP388(SPI)
+
+•	builtin RAMTRON(SPI)
+
+•	microSD card slot
+
+•	5 UARTs
```

Comment:
```suggestion
•	5 UARTs, two with RTS/CTS flow control
```

Reply:
I have modified and added default.param.